src/services/downloadManager.ts

```diff
@@ -131,13 +136,14 @@ class DownloadManager {
         }
         if (
             file.metadata.fileType === FILE_TYPE.IMAGE ||
-            file.metadata.fileType === FILE_TYPE.LIVE_PHOTO
+            file.metadata.fileType === FILE_TYPE.LIVE_PHOTO ||
+            file.metadata.fileType === FILE_TYPE.VIDEO
```

Comment:
Sorry, don't have full context but why we are adding VIDEO here now? Also, if we are adding VIDEO, why not just remove this condition and use it for all types?

Reply:
yes would be same....we can discuss this on call